Sources/PrimerSDK/Classes/User Interface/Root/PrimerNavigationBar.swift

```diff
@@ -24,16 +24,44 @@ class PrimerNavigationBar: PrimerView {
         didSet {
             rightBarButton?.tintColor = theme.text.system.color
             rightBarButton?.setTitleColor(theme.text.system.color, for: .normal)
-            rightBarButton?.frame = CGRect(
-                x: 0, y: 0, width: rightView.bounds.size.width, height: rightView.bounds.size.height
-            )
```

Comment:
This code was not working, at the moment of the frame setting, right view was not rendered yet... use Autolayout instead of frame setting